### PR TITLE
DB Migration

### PR DESCRIPTION
**Pull request by gnorbsl**
- Changed wishcardTo field to belongsTo to avoid confusion
- Added migrationsscript that fills belongsTo on existing databases

### CODE REVIEW

**Review by Jacobjeevan — APPROVED** [review]
:+1: While you are at it, do you also want to take care of the status field as well?

Ensure all existing wishcards have a status of published on prod (I think Stacey was manually setting this, but in case she missed any). 

We can then revert GET (/wishcard) route back to returning only published and donated cards.

File: server/routes/wishCards.js [hunk 1]
```diff
@@ -369,11 +369,11 @@ router.get('/:id', redirectLogin, getByIdValidationRules(), validate, async (req
...
-   
-    // NOTE: 
+
+    // NOTE:
     // this agency object is returning undefined and breaking frontend
-    const agency = wishcard.wishCardTo;
-    
+    const agency = wishcard.belongsTo;
+
     console.log("test print");
```
Comment (Jacobjeevan) [inline server/routes/wishCards.js:377]: I think someone added this for testing (can be removed now with seeder + migrations)

Comment (gnorbsl) [reply server/routes/wishCards.js]: done

**Comment by gnorbsl** [comment]
> 👍 While you are at it, do you also want to take care of the status field as well?
> 
> Ensure all existing wishcards have a status of published on prod (I think Stacey was manually setting this, but in case she missed any).
> 
> We can then revert GET (/wishcard) route back to returning only published and donated cards.

added

**Pull request merged**